src/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -48,6 +50,15 @@ public abstract partial class PipeReader
         /// </remarks>
         public abstract void AdvanceTo(SequencePosition consumed, SequencePosition examined);
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public virtual Stream AsStream()
+        {
+            return _stream ?? (_stream = new PipeReaderStream(this));
```

Comment:
```suggestion
            return _stream ??= new PipeReaderStream(this));
```

It is C#8 already :sunglasses: